test/sharness/t0050-block.sh

```diff
@@ -209,4 +209,16 @@ test_expect_success "no panic in output" '
   test_expect_code 1 grep "panic" stat_out
 '
 
+test_expect_success "can set multihash type and length on block put without format" '
+  HASH=$(echo "foooo" | ipfs block put --mhtype=sha3 --mhlen=16)
```

Comment:
16 will be too short of a hash length after cid changes come through, please change it to 20.

Reply:
done